docs/about/release-notes.md

```diff
@@ -25,6 +25,25 @@ The current and past members of the MkDocs team.
 
 ### Major Additions to Development Version
 
+#### Path Based Settings are Relative to Configuration File (#543)
+
+Previously any relative paths in the various configuration options were
+resolved relative to the current working directory. They are now resolved
+relative to the configuration file. As the documentation has always encouraged
```

Comment:
@waylan I couldn't find it anywhere in the docs, but what about for inherited config files? Are the paths in those inherited config files relative to the parent config file or the child config file?

Reply:
Please don't necro-bump and especially don't tag people while doing so.

The answer is here
https://github.com/mkdocs/mkdocs/blob/ae4a639f6e6fa3648e033f8d2f2cb7679886887b/docs/user-guide/configuration.md?plain=1#L1047

Reply:
(you can ask a question as a Discussion instead)